makefile

```diff
@@ -387,6 +385,15 @@ scramble:
 	python cmd/scrambled-eggs/scramble.py $(file) $(multiarg)
 
 # mongodb utility targets
+mongotoolsDir := mongotools
+run-mongotools = $(if $(wildcard $(mongotoolsDir)), ./$(mongotoolsDir)/$(1), $(1))
```

Comment:
`:=` is preferred in Make unless you have a specific need for `=` because `=` can lead to infinite recursion.

Reply:
If I use `:=` it won't expand the arg param `$(1)`.